src/helm/proxy/clients/openai_client.py

```diff
@@ -44,6 +49,22 @@ def make_request(self, request: Request) -> RequestResult:
                 "input": request.prompt,
                 "engine": request.model_engine,
             }
+        elif self._is_chat_model_engine(request.model_engine):
+            raw_request = {
+                "model": request.model_engine,
+                # For now, put the whole prompt in a single user message.
+                # TODO: Support ChatML for creating multiple messages with different roles.
+                # See: https://github.com/openai/openai-python/blob/main/chatml.md
+                "messages": [{"role": "user", "content": request.prompt}],
+                "temperature": request.temperature,
+                "top_p": request.top_p,
+                "n": request.num_completions,
+                # TODO: Setting stop causes the server to return a HTTP 500 error.
+                # "stop": request.stop_sequences or None,  # API doesn't like empty list
+                "max_tokens": request.max_tokens + 1,  # Uhhh
```

Comment:
What is this `+1` for?

Reply:
Removing the +1 for now.

The special roles take up tokens, so if you want 1 token, you actually have to ask 2 tokens. But I'd rather handle this elsewhere.